app/app.ts

```diff
@@ -18,7 +18,7 @@ import {loginConfiguration} from './config/constants';
 let providers = [
   HTTP_PROVIDERS,
   provide(OvhRequestService, {
-    useFactory: (http) => {
+    useFactory: http => {
```

Comment:
Alright after review all your changes, I want to keep this guideline. In fact I always want to put () ever if there are just 1 arg. I found that more clear. Could you let the () please ?